src/hooks/useTasks.js

```diff
@@ -15,67 +15,75 @@ export default function useTasks() {
 }
 
 function useOpenTasks(tasks, now, courtSettings) {
-  const currentRoundPhases = useMemo(() => {
+  const convertedTasks = useMemo(() => {
     if (!tasks) {
       return null
     }
-    return tasks.map(t =>
-      getAdjudicationPhase(t.dispute, t, now, courtSettings)
-    )
+    return tasks.map(task => ({
+      ...task,
+      ...getAdjudicationPhase(task.dispute, task, now, courtSettings),
+    }))
   }, [courtSettings, now, tasks])
 
-  const currentRoundPhasesKey = currentRoundPhases
-    ? currentRoundPhases
+  const convertedTasksPhasesKey = convertedTasks
+    ? convertedTasks
         .map(phase => DisputesTypes.convertToString(phase.phase))
         .join('')
     : null
 
   return useMemo(() => {
+    if (!tasks) {
```

Comment:
```suggestion
    if (!convertedTasks) {
```

We don't rely on `tasks` anymore in this second `useMemo()`, so we should only operate with data we're actually going to use (rather than a transient dependency).